airflow/jobs/scheduler_job_runner.py

```diff
@@ -1389,6 +1388,11 @@ def _schedule_dag_run(
             self.log.error("Couldn't find DAG %s in DAG bag or database!", dag_run.dag_id)
             return callback
         if not dag_model:
+            self.log.info(
+                "DAG %s scheduling was skipped, "
+                "probably because DAG was None due to skip_locked in with_row_locks",
```

Comment:
A regular Airflow user does not likely know what _due to skip_locked in with_row_locks_ mean (they are functions only people implementing Airflow would use). It would be a good idea to use a more generic wording here.

Reply:
I felt exactly the same way.
I have corrected the log message https://github.com/apache/airflow/pull/31414/commits/fe6df83a84fa7c64799afeea8165d2bcfbf833ce